WcaOnRails/app/views/static_pages/disclaimer.html.erb

```diff
@@ -4,9 +4,10 @@
   <h1><%= yield(:title) %></h1>
 
   <ul>
-    <li>The World Cube Association (WCA) works to verify that appropriate WCA Regulations are followed for the competition portion of WCA-recognized competitions.</li>
-    <li>However, all other aspects of competitions, including (but not limited to) financial matters and competition websites, are solely the responsibility of the competition organizers and not the WCA.</li>
+    <li>The World Cube Association (WCA) works to verify that appropriate WCA Regulations are followed for the competition portion of WCA Competitions.</li>
+    <li>However, all other aspects of competitions, including (but not limited to) financial matters and competition websites, are solely the responsibility of the competition organizers and not the WCA's.</li>
```

Comment:
I don't understand this change. Adding the apostrophe s makes it feel like a sentence fragment. "... not the WCA's <what>"?

Reply:
This should not have been changed.